### ISSUE TITLE
.addText needs some margin bottom or it crops

### ISSUE THREAD

**Issue by ugran** [andriyoganp/blue_print_pos#13]
Hello, First of all I'd like to state that this is a lifesaver project - The only flutter package with which I was able to print from iOS on a phomemo printer I bought from amazon.

Only problem is that when using **.addText** the printed text is cut on the bottom (.p needs some margin-bottom).

**.addLeftRightText** prints nice.

Also, are you going to maintain this project & update it to null safety?

**Comment by andriyoganp** [comment]
Yeah, it happens on the last print. Use .`addSpacer()` in last process. Because .addText use tag `<div>` and `.addLeftRightText` use tag `<p>`. I will figure it out later. Thanks

**Comment by hamzamon** [comment]
Thank you for the project, there is a big gap between upper portion and qr code even after feedCount: 0, how reduce this unwanted space?

**Comment by hamzamon** [comment]
It displays all available devices in the scan result, how can it only display bluetooth printer devices?
